mathesar/rpc/analytics.py

```diff
@@ -105,3 +106,9 @@ def view_report() -> AnalyticsReport:
     """
     report = prepare_analytics_report()
     return AnalyticsReport.from_dict(report)
+
+
+@mathesar_rpc_method(name="analytics.upload_feedback", auth="login")
+def upload_feedback(message: str) -> None:
```

Comment:
Please add an args section for `message` in the docstring for our docs.

Reply:
good catch.